pkgs/applications/misc/urlscan/default.nix

```diff
@@ -1,30 +1,43 @@
 { lib
-, python3Packages
 , fetchFromGitHub
+, python3
 }:
 
-python3Packages.buildPythonApplication rec {
+python3.pkgs.buildPythonApplication rec {
   pname = "urlscan";
-  version = "0.9.10";
+  version = "1.0.0";
+  format = "pyproject";
 
   src = fetchFromGitHub {
     owner = "firecat53";
     repo = pname;
-    rev = version;
-    hash = "sha256-lCOOVAdsr5LajBGY7XUi4J5pJqm5rOH5IMKhA6fju5w=";
+    rev = "refs/tags/${version}";
+    hash = "sha256-IvCdc4f784hBM+TEa0zIACz/1/FUnEpGxHUGiS85tt8=";
   };
 
-  propagatedBuildInputs = [
-    python3Packages.urwid
+  SETUPTOOLS_SCM_PRETEND_VERSION = version;
+
+  nativeBuildInputs = with python3.pkgs; [
+    hatchling
+    hatch-vcs
+    setuptools-scm
```

Comment:
this is not listed in [pyproject.toml](https://github.com/firecat53/urlscan/blob/0048524efa905d5966a50c9376f4989432135773/pyproject.toml#L2), we can remove it since it is a transitive dependency of `hatch-vcs`
```suggestion
```

Reply:
Removed, thanks.